core/modules/catalog/types/ProductConfiguration.ts

```diff
@@ -0,0 +1,10 @@
+export interface Configuration {
```

Comment:
i'd change it to ProductOption. Configuration is more reserved to application Configuration.